web/src/user_deactivation_ui.ts

```diff
@@ -61,6 +67,7 @@ export function confirm_deactivation(
                 admin_email: people.my_current_email(),
                 realm_url,
                 realm_name,
+                message_delete_actions: get_message_delete_actions(),
```

Comment:
I guess this can directly call `Object.values(settings_config.message_delete_action_values)`. Or better to just have `settings_config.message_delete_action_values` as a list instead of an object.